pyproject.toml

```diff
@@ -24,8 +24,10 @@ dependencies = [
     "opencv-python>=4.9.0.80",
     "pillow>=10.3.0",
     "pydub>=0.25.1",
-    "soundpython>=0.1.9",
+    "soundpython>=0.1.11",
     "tqdm>=4.66.3",
+    "openai-whisper>=20240930",
+    "numba>=0.61.0",
```

Comment:
Imo if we're creating new `videopython.generation` submodule we should also greate new dependency group for `understanding`.

I'm not sure if should go this route tho, alternative would be doing `videopython.ai.generation` and `videopython.ai.understanding` modules and creating one shared dependency group betweeon those. What do you think?

Reply:
Yea, I think we should do one `ai` dependency group instead of creating another one for understanding. Do you want to change the library structure to have this ai parent folder as well?

Reply:
Yeah, let's do `videopython.ai.generation` and `videopython.ai.understanding` with only one `ai` dependency group in pyproject 😎